### ISSUE TITLE
Error: unrecognised flag

### ISSUE THREAD

**Issue by drzero42** [loophp/flake-lock-update-workflow#176]
I have been using your Github Action for a while and have been very happy with it.

Today I started getting this error: `error: unrecognised flag '--commit-lockfile-summary'`.

It looks like it is because that flag was recently renamed [here](https://github.com/NixOS/nix/pull/10691).

**Comment by drupol** [comment]
Hello!

Glad to see it is useful :)

Do you mind send a quick PR to fix the issue?

Thanks!

**Comment by drzero42** [comment]
I tested a bit further, and it did not recognize `--commit-lock-file-summary` either. Then I tried with `--commit-lockfile-summary` again, and now it works. Reading the actual code change in the PR mentioned, I now see that it actually does not remove the old argument, it just adds the new spelling as an additional way to trigger the same.
I have no idea why I got these errors, but now it seems to work correctly again :shrug:. Closing the issue.
